_posts/2017-11-14-django-girls-bangalore-2017.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: post
+title: "Django girls Bangalore 2017"
+date: "2017-11-14 08:01:25 +0530"
+tag:
+  - django
+  - python
+  - djangogirls
+---
+
+TL;DR
+
+Last Sunday, Django girls Bangalore organized a hands-on session of Django. This
+is small event report from my side.
+
+Django girls is not for profit initiative led by Ola Sitarska and Ola Sendecka.
+Such movement helps women to learn the skills of website development using the
+well-known web-framework Django. This community is backed by organizers from
+many countries of this world. Organizations like The Python Software Foundation,
+Github, DjangoProject and much more are funding Django girls.
+
+Django girls Bangalore chapter was organized by Sourav Singh and Kumar Anirudha.
+It was my second time mentoring for the Django girls event. First was for
+Ahmedabad chapter. The venue was sponsored by HackerEarth. X male and 2 women
```

Comment:
What is X male? From X-men movie?

Reply:
The numbers are pending to arrive from the organizers. I have added a note regarding this in a PR description. Thanks!

Reply:
In any case. X "male" and Y "women" doesn't feel okay. Either male/female, or men/women.